gradle/validation/forbidden-apis.gradle

```diff
@@ -49,11 +49,13 @@ allprojects { prj ->
           logger.debug("Signature file omitted (does not exist): ${sig}")
         }
       }
-      
+
       // commons-io is special: forbiddenapis has a versioned bundledSignature.
-      bundledSignatures += resolvedMods
-        .findAll { id -> id.group == 'commons-io' && id.name == 'commons-io' }
-        .collect { id -> "${id.name}-unsafe-${id.version}" as String }
+//      bundledSignatures += resolvedMods
+//        .findAll { id -> id.group == 'commons-io' && id.name == 'commons-io' }
+//        .collect { id -> "${id.name}-unsafe-${id.version}" as String }
+      // Hack until forbiddenapis supports commons-io 2.12.0
+      bundledSignatures += ['commons-io-unsafe-2.11.0']
```

Comment:
This now adds commons-io to all Solr modules, although it may not be part of dependencies. This should not hurt, but could cause failures if we have a Solr module that does not refer to commons-io.

I'd change only line 56 to be:
```groovy
.collect { id -> "${id.name}-unsafe-2.11.0" as String }
```

This ensures the forbiddenapis are only added if the compilation unit depends on commons-io.

Maybe keep the existing line commented.

Reply:
Well, I see commons-IO yesterday released 2.13.0 to fix some regressions in 2.12. In fact I think the test failure on this PR may be due to that.. I'll re-purpose this to upgrade to 2.13

Reply:
I have seen your PR for commons-io. I will update this.

Theres also another forbiddenapis issue to make the version parsing a bit better for commons-io. So whenever a newer version is passed to the bundled signatures it will fallback to the last recent version before the requested one.

This should solve issues like this in future. I just have no time before bbuzz.... (my talk is still mostly unprepared)